src/util.h

```diff
@@ -31,33 +31,24 @@
 #include <unordered_set>
 #include <vector>
 
-#include <boost/signals2/signal.hpp>
 #include <boost/thread/condition_variable.hpp> // for boost::thread_interrupted
 
 // Application startup time (used for uptime calculation)
 int64_t GetStartupTime();
 
-/** Signals for translation. */
-class CTranslationInterface
-{
-public:
-    /** Translate a message to the native language of the user. */
-    boost::signals2::signal<std::string (const char* psz)> Translate;
-};
-
-extern CTranslationInterface translationInterface;
-
 extern const char * const BITCOIN_CONF_FILENAME;
 extern const char * const BITCOIN_PID_FILENAME;
 
+/** Translate a message to the native language of the user. */
+const extern std::function<std::string(const char*)>* G_TRANSLATION_FUN;
```

Comment:
How about
```cpp
const extern std::function<std::string(const char*)> G_TRANSLATION_FUN;
```
And below (L51) use https://en.cppreference.com/w/cpp/utility/functional/function/operator_bool?

Reply:
Done